EXCEPTION DELETE
Encountered exception(s)

- Path not found: <dir>/d5/f3
CallStack (from HasCallStack):
  throwCS, called at src/SafeRm/Data/PathData.hs:196:14 in safe-rm-0.1-inplace:SafeRm.Data.PathData
- Path not found: <dir>/d5/f4
CallStack (from HasCallStack):
  throwCS, called at src/SafeRm/Data/PathData.hs:196:14 in safe-rm-0.1-inplace:SafeRm.Data.PathData


CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:111:5 in safe-rm-0.1-inplace:SafeRm

LOGS DELETE
[2020-05-31 12:00:00][functional.delete][Debug] Trash home: <dir>/d5/.trash
[2020-05-31 12:00:00][functional.delete.deleting][Debug] MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f1"}, originalPath = MkPathI {unPathI = "<dir>/d5/f1"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.delete.deleting][Debug] MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f2"}, originalPath = MkPathI {unPathI = "<dir>/d5/f2"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.delete.deleting][Warn] Path not found: <dir>/d5/f3
CallStack (from HasCallStack):
  throwCS, called at src/SafeRm/Data/PathData.hs:196:14 in safe-rm-0.1-inplace:SafeRm.Data.PathData
[2020-05-31 12:00:00][functional.delete.deleting][Warn] Path not found: <dir>/d5/f4
CallStack (from HasCallStack):
  throwCS, called at src/SafeRm/Data/PathData.hs:196:14 in safe-rm-0.1-inplace:SafeRm.Data.PathData
[2020-05-31 12:00:00][functional.delete.deleting][Debug] MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f5"}, originalPath = MkPathI {unPathI = "<dir>/d5/f5"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.delete][Info] Deleted: <dir>/d5/f5, <dir>/d5/f2, <dir>/d5/f1
[2020-05-31 12:00:00][functional][Error] Encountered exception(s)

- Path not found: <dir>/d5/f3
CallStack (from HasCallStack):
  throwCS, called at src/SafeRm/Data/PathData.hs:196:14 in safe-rm-0.1-inplace:SafeRm.Data.PathData
- Path not found: <dir>/d5/f4
CallStack (from HasCallStack):
  throwCS, called at src/SafeRm/Data/PathData.hs:196:14 in safe-rm-0.1-inplace:SafeRm.Data.PathData


CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:111:5 in safe-rm-0.1-inplace:SafeRm


TERMINAL LIST
Type:      File
Name:      f1
Original:  <dir>/d5/f1
Created:   2020-05-31 12:00:00 

Type:      File
Name:      f2
Original:  <dir>/d5/f2
Created:   2020-05-31 12:00:00 

Type:      File
Name:      f5
Original:  <dir>/d5/f5
Created:   2020-05-31 12:00:00 
Entries:      3
Total Files:  3
Log size:     0.00B
Size:         20.00B 


